Docs update (#41)
epiphany-platform/m-azure-basic-infrastructure: README.md

Epiphany Module: Azure Basic Infrastructure

AzBI module is responsible for providing basic cloud resources (eg. resource groups, virtual networks, subnets, virtual machines ect.) which will be used by upcoming modules.

# Basic usage

## Requirements

Requirements are listed in a separate [document](docs/REQUIREMENTS.md).

## Build image

In main directory, run:

  ```shell
  make build
  ```

  Note: This command uses the default VERSION variable.

or directly using Docker:

  ```shell
  cd m-azure-basic-infrastructure/
  docker build --tag epiphanyplatform/azbi:latest .
  ```

> Note: Re-run the above commands will overwrite your docker image. To bypass that, specify a different tag name or VERSION variable.

Reviewer: ```suggestion
Note: Re-run the above commands will overwrite your existing docker image (if exists). To bypass that, specify a different tag name
```

## Run module

* Create a shared directory:

  ```shell
  mkdir /tmp/shared
  ```

  This 'shared' dir is a place where all configs and states will be stored while working with modules.

* Generate ssh keys in: /tmp/shared/vms_rsa.pub

  ```shell
  ssh-keygen -t rsa -b 4096 -f /tmp/shared/vms_rsa -N ''
  ```

* Initialize AzBI module:

  ```shell

>   docker run --rm -v /tmp/shared:/shared -t epiphanyplatform/azbi:latest init M_VMS_COUNT=2 M_PUBLIC_IPS=true M_NAME=epiphany-modules-test

Reviewer: ```suggestion
  docker run --rm -v /tmp/shared:/shared -t epiphanyplatform/azbi:latest init M_VMS_COUNT=2 M_PUBLIC_IPS=true M_NAME=epiphany-modules-test
  
Note: Use image's tag according to tag generated in build step.
```
